process_jartic_simple.py

```
@@ -0,0 +1,163 @@
+    if not archives:
+        print(f"❌ No archives found in {args.cache_dir}")
+        return
+    
+    print(f"\n🚗 Simple JARTIC Processor")
+    print("="*60)
+    print(f"Archives found: {len(archives)}")
+    print(f"Date range: {args.start} to {args.end}")
+    
+    # Prepare output
+    storage = DataStorage()
+    timestamp = datetime.now().strftime('%Y%m%d_%H%M%S')
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)